### DISCUSSION TITLE
Add global CSS classes to <html>

### DISCUSSION THREAD

**Discussion by gouku** [facebook/docusaurus#7072] [Q&A] [answered]
Is there any easy way to add global CSS classes to `<html>` or `<body>` so I can customize it based on different page, post, etc.

Thanks!

**Comment by Dannyholt** [comment, marked as answer]
See this doc here - https://docusaurus.io/docs/markdown-features/head-metadata

**Reply by gouku** [reply]
Thanks! This should work but I don't know how to apply these metadata to Pages written in JavaScript? Any examples would be appreciated.

**Reply by gouku** [reply]
Ok never mind there's [another doc](https://docusaurus.io/docs/docusaurus-core#head) how to use `<Head>` component in JavaScript pages. Thanks!
